website/docs/components/models/filesystem.md

```diff
@@ -1,41 +1,58 @@
 ---
-title: 'Filesystem'
+title: 'Filesystem Models'
 description: 'Instructions for using models hosted on a filesystem with Spice.'
 sidebar_label: 'Filesystem'
 sidebar_position: 5
 ---
 
-To use a model hosted on a filesystem, specify the path to the model file in the `from` field.
-
-Supported formats include ONNX for traditional machine learning models and GGUF, GGML, and SafeTensor for large language models (LLMs).
-
-### Example: Loading an ONNX Model
+To use a model hosted on a filesystem, specify the path to the model file or folder in the `from` field:
 
 ```yaml
 models:
-  - from: file://absolute/path/to/my/model.onnx
-    name: local_fs_model
+  - from: file:models/llms/llama3.2-1b-instruct/
+    name: llama3
+    params:
+      model_type: llama
+      
 ```
 
-### Example: Loading a GGUF Model
+Supported formats include GGUF, GGML, and SafeTensor for large language models (LLMs) and ONNX for traditional machine learning (ML) models.
```

Comment:
Thought about moving this to limitations, but decided to keep it here as it is important information that should be on top IMO